traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -2123,8 +2123,8 @@ func UpdateServerProfilesForV4(id int, profile []string, tx *sql.Tx) error {
 	return nil
 }
 
-// UpdateServerProfileTableForV2V3 updates CommonServerPropertiesV40 struct and server_profile table via Update (server) function for API v2/v3.
-func UpdateServerProfileTableForV2V3(id *int, newProfileId *int, origProfile string, tx *sql.Tx) error {
+// UpdateServerProfileTableForV3 updates CommonServerPropertiesV40 struct and server_profile table via Update (server) function for API v3.
+func UpdateServerProfileTableForV3(id *int, newProfileId *int, origProfile string, tx *sql.Tx) error {
```

Comment:
nit: Comment below also references V2. Line 2141

Reply:
Removed reference in fc419ac2ad